bin/inspect_all_known_projects.py

```diff
@@ -15,7 +15,7 @@
 DIR = Path(__file__).parent.resolve()
 
 
-def parse(contents: str) -> Optional[str]:
+def parse(contents: str) -> str | None:
```

Comment:
Why is it only rewriting things here? There's plenty of other use of `Optional[Foo]`?